app/views/expenses/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<%# path = app/views/expenses/new.html.erb %>
+<div class="categories-index">
+  <div class="header">
+    <span><a href="<%= category_path(@expense.category_id) %>" class="back-arrow">←</a></span>
```

Comment:
* This error reported by the previous CR still persist, when i click on the add expense button i get the error below 👇 Kindly fix it 
![image](https://github.com/noelfoka/capstone_budget_app/assets/52497246/1bd42b2a-536b-49f7-bd4c-89084526456e)

Reply:
I need help